blog/ansible-library.md

```diff
@@ -0,0 +1,177 @@
+The following playbook show a mix of `ID` and a [Gremlin](/documentation/api-gremlin) expression to link the switch
+port to a host interface. 
+
+{% highlight yaml %}
+- name: Skydive Ansible
+  connection: local
+  hosts: localhost
+  tasks:
+
+    - name: Create TOR
+      skydive_node:
+        name: 'TOR'
+        type: "fabric"
+        metadata:
+          Model: My favorite Switch brand
+          ManagementIP: 10.0.0.1
+      register: tor_result
+
+    - name: Create port
+      skydive_node:
+        name: 'PORT'
+        type: 'fabric'
+      register: port_result
+    
+    - name: Link TOR and port  
+      skydive_edge:
+        node1: "{{ tor_result.UUID }}"
+        node2: "{{ port_result.UUID }}"
+        relation_type: ownership
+
+    - name: Link port and eth0  
+      skydive_edge:
+        node1: "{{ port_result.UUID }}"
+        node2: "G.V().Has('Name', 'enp0s25')"
+        relation_type: layer2
+{% endhighlight %}
+
+This gives the following topology.
+
+<p>
+  <a href="/assets/images/blog/ansible-lib-2.png" data-lightbox="WebUI-1" data-title="Skydive WebUI">
+    <img src="/assets/images/blog/ansible-lib-2.png"/>
+  </a>
+</p>
+
+
+## Ansible, Skydive and LLDP
+
+Finally the following example will be a bit more concrete. It uses `LLDP` to retrieve physical topology information
+and the `Skydive` module to inject them into the `Skydive` topology.
+
+{% highlight yaml %}
+- name: Skydive Ansible
+  connection: local
+  hosts: localhost
+  tasks:
+
+  - name: Gather information from lldp
+    lldp:
+    become: true
+
+  - name: Create TOR
+    skydive_node:
+      name: "TOR - {{ lldp[item]['chassis']['name'] }}"
+      type: "fabric"
+      id: "{{ lldp[item]['chassis']['name'] }}:{{ lldp[item]['chassis']['mac'] }}"
+      metadata:
+        MAC: "{{ lldp[item]['chassis']['mac'] }}"
+        Description: "{{ lldp[item]['chassis']['descr'] }}"
+        Address: "{{ lldp[item]['chassis']['mgmt-ip'] }}"
+    register: tors
+    with_items: "{{ lldp.keys() }}"
+
+  - name: Create port
+    skydive_node:
+      name: "{{ lldp[item]['port']['descr'] }}"
+      type: "fabric"
+      id: "{{ lldp[item]['port']['descr'] }}:{{ lldp[item]['port']['mac'] }}"
+      metadata:
+        MAC: "{{ lldp[item]['port']['mac'] }}"
+    register: ports
+    with_items: "{{ lldp.keys() }}"
+    
+  - name: Link port to TOR
+    skydive_edge:
+      node1: "{{ item.0.UUID }}"
+      node2: "{{ item.1.UUID }}"
+      relation_type: ownership
+    with_together:
+      - "{{ tors.results }}"
+      - "{{ ports.results }}"
+
+  - name: Link port to host interface
+    skydive_edge:
+      node1: "{{ item.0.UUID }}"
+      node2: "G.V().Has('Type', 'host').Out().Has('Name', '{{ item.1 }}')"
+      relation_type: ownership
+    with_together:
+      - "{{ ports.results }}"
+      - "{{ lldp.keys() }}"
+{% endhighlight %}
+
+## conclusion
+
+As we saw the `Skydive` module allow to leverage the `Ansible` ecosystem. This makes really makes sense
```

Comment:
[the Network Automation Ansible ecosystem](https://www.ansible.com/integrations/networks)

Reply:
Remove line :
" This makes really makes sense thinking about all the networking existing module."